src/NetMQ/Core/Utils/OpCode.cs

```diff
@@ -11,6 +11,14 @@ internal static class Opcode
 
         public static bool Open()
         {
+            //Look for an environment variable: "NETQM_SUPPRESS_RDTSC" with a value of "TRUE"
+            // The application can set this environment variable when this code is running in a system where
+            // it is not desirable to read the processor's time stamp counter.
+            // While this is supported in modern CPUs, the technique used for allocating executable memory, copying OP Code
+            // for the read of the time stamp and invoking the OP Code can be detected as Malware by some anti-virus vendors.
```

Comment:
Please include a link to the original issue here so that we can more easily see the motivation for this feature in future:

```suggestion
            // for the read of the time stamp and invoking the OP Code can be detected as Malware by some anti-virus vendors.
            // https://github.com/zeromq/netmq/issues/1071
```